tests/test_generate_data.py

```diff
@@ -65,3 +66,79 @@ def test_params_same_from_sim_data(self):
                  grouped_data['n_custs'])
         assert ((np.array(list(bbgb_params.values())) - np.array(
             bbtf._unload_params('alpha', 'beta', 'gamma', 'delta'))) < 0.1).all()
+
+
+bg_params = OrderedDict([('T', 100), ('r', 0.24), ('alpha', 4.41),  ('a', 0.79),
+                         ('b', 2.43), ('size', 800)])
+bg_params_with_freq = OrderedDict([('T', 100), ('r', 0.24), ('alpha', 4.41),  ('a', 0.79),
+                         ('b', 2.43), ('size', 800), ('freq', 'h')])
+
+
+class TestTransactionalDataGeneration():
+
+
+    # test using T as int
+    def test_beta_geometric_nbd_model_transactional_data_int_T(self):
+        np.random.seed(188898)
+        summary_original = beta_geometric_nbd_model(**bg_params)[['frequency', 'recency', 'T']]
+        np.random.seed(188898)
```

Comment:
why twice?

Reply:
Because using it twice to make sure both functions after these two random seeds can generate results with the same seed. I find random seed can only be applied to the next function.